cpp/cmake_modules/FindorcAlt.cmake

```diff
@@ -29,6 +29,7 @@ endif()
 find_package(orc ${find_package_args})
 if(orc_FOUND)
   set(orcAlt_FOUND TRUE)
+  set(orcAlt_VERSION ${orc_VERSION})
```

Comment:
Does this work with conan？

Reply:
I hope so...
Do you know conan's ORC recipe provide valid(?) CMake package for ORC?

(BTW, is there a plan that ORC provides its CMake package?)

Reply:
> Do you know conan's ORC recipe provide valid(?) CMake package for ORC?

I suppose it does. It was added by me: https://github.com/conan-io/conan-center-index/blob/master/recipes/orc/all/conanfile.py#L145

> is there a plan that ORC provides its CMake package?

I'm not sure if I understand the question correctly, do you mean the ORC community provides an official FindOrc.cmake file or something similar?

Reply:
OK, let me investigate how does it work and add it to ORC.